android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -136,6 +136,9 @@
     private boolean customTabsSupported;
     private CustomTabsServiceConnection customTabsServiceConnection;
     private CustomTabsClient customTabsClient;
+    private Runnable runnable;
+    private Handler handler = new Handler();
```

Comment:
Any special reason for initialising the handler outside of the method it is used?